src/pages/question/questionBuilder/useNode.js

```diff
@@ -68,46 +67,36 @@ export default function useNodePanels() {
     setSearchTerm(entityNameDisplay(entry.name));
     setName(entityNameDisplay(entry.name));
     if (entry.curie) {
-      setCurie(entry.curie);
+      setCurie([entry.curie]);
```

Comment:
Can we be certain that entry.curie will always be a string and not an array here? I think this is fine for now, but something to keep in mind.

Reply:
Entries come from the ranker API which says that the resulting curie will be a string. I think it makes sense to "trust" whatever API we are calling instead of handling the possibility that it is not behaving as documented.

Reply:
👍